share/functions/__fish_whatis_current_token.fish

```diff
@@ -1,19 +1,17 @@
 # This function is typically bound to Alt-W, it is used to list man page entries
 # for the command under the cursor.
 function __fish_whatis_current_token -d "Show man page entries related to the token under the cursor"
-    set tok (commandline -pt)
+    set -l tok (commandline -pt)
 
-    if test $tok[1]
```

Comment:
I don't know what's the best thing to do when there is no word at the cursor.
This looks fine, or we can return (and perhaps send an alert like `__fish_man_page`).

Reply:
The current implementation of Alt+W simply uses this if-check, so I'm in favor of leaving it how it is for now.